src/entities/endpoint.entity.ts

```diff
@@ -1,8 +1,7 @@
 import { SharedEntity } from '../common/model/sharedEntity';
 import { BeforeInsert, Column, Entity } from 'typeorm';
 import { HttpMethod } from '../common/enums/httpMethods.enum';
-import { ReqBody } from '../endpoints/interface/endpoint.interface';
-const urlEncoode = require('urlencode')
```

Comment:
im curious, why the common Js way of importing??

Reply:
The background is red, that means it was removed.